src/components/SkillCard/SkillCard.scss

```diff
@@ -0,0 +1,25 @@
+@use '../../sass/abstracts/variables';
+
+.skillCard {
+  background-size: cover;
+  min-width: 300px;
+  min-height: 300px;
```

Comment:
we will have to fix these sizes with grid instead..

Reply:
removing the min-width and min-height displaces the text for the cards

Reply:
Used flex-box as suggested to align the text and image inside the skill card